src/library_dylink.js

```diff
@@ -201,9 +201,30 @@ var LibraryDylink = {
 #endif
     ;
   },
+  $setGOTEntry__deps: ['$addFunction', '$getFunctionAddress'],
+  $setGOTEntry: (symName, value) => {
+    const entry = GOT[symName];
```

Comment:
It looks like both calls sites already have looked up the entry so perhaps we can pass the entry here too to avoid this extra lookup?